torch/functional.py

```diff
@@ -347,9 +347,18 @@ def einsum(equation, *operands):
         run faster or consume less memory. Projects like opt_einsum (https://optimized-einsum.readthedocs.io/en/stable/)
         can optimize the formula for you.
 
+    .. note::
+
+        There's an alternative way of calling :func:`torch.einsum` by providing each input operand followed by its
```

Comment:
What do you think about separating the docs update so we can get higher code velocity and focus on the docs in a follow-up PR?

For now, for example, this note could read, "As of PyTorch 1.9 :func:\`torch.einsum\` also supports the sublist format (see examples below)." ?

Reply:
Sure, I'll do that.

Reply:
PR adding doc here https://github.com/pytorch/pytorch/pull/57038